azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/__init__.py

```diff
@@ -4,29 +4,33 @@
 # license information.
 # --------------------------------------------------------------------------
 from azure.monitor.opentelemetry.distro.util import get_configurations
-from azure.monitor.opentelemetry.exporter import AzureMonitorTraceExporter
+from azure.monitor.opentelemetry.exporter import (
+    ApplicationInsightsSampler,
+    AzureMonitorTraceExporter,
+)
 from opentelemetry import trace
 from opentelemetry.sdk.resources import Resource
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import BatchSpanProcessor
 from opentelemetry.semconv.resource import ResourceAttributes
 
 
-def configure_opentelemetry(**kwargs):
+def configure_azure_monitor(**kwargs):
     """
     This function works as a configuration layer that allows the
     end user to configure OpenTelemetry and Azure monitor components. The
-    configuration can be done via environment variables or
-    via arguments passed to this function. Each argument has a 1:1
-    correspondence with an environment variable.
+    configuration can be done via arguments passed to this function.
     """
 
     configurations = get_configurations(**kwargs)
-    connection_string = configurations["connection_string"]
```

Comment:
Why remove conn str?

Reply:
We are taking advantage of passing in the whole `kwargs` into the exporter so no need to explicitly extract the connection string.